src/test/fuzz/descriptor_parse.cpp

```diff
@@ -11,6 +11,7 @@
 void initialize()
 {
     static const ECCVerifyHandle verify_handle;
+    ECC_Start();
     SelectParams(CBaseChainParams::REGTEST);
```

Comment:
```suggestion
    SelectParams(CBaseChainParams::MAIN);
```

or alternatively change this to MAIN

Reply:
done